source/plugins/go/input/cmetrics_windows.go

```diff
@@ -0,0 +1,315 @@
+//go:build windows

```

Comment:
any progress on the PR on cmetrics to include this?

Reply:
No progress...I think they mentioned something in slack...Will try to address that